types/styled-system/index.d.ts

```diff
@@ -8,6 +8,7 @@
 //                 Matthieu Vachon <https://github.com/maoueh>
 //                 Adam Lavin <https://github.com/lavoaster>
 //                 Joachim Schuler <https://github.com/jschuler>
+//                 Adam Misiorny <https://github.com/adam187>
```

Comment:
Just so you know, by putting your name here you're signing up for receiving and maintaining these types as in future PR's and issues you will be tagged for help in resolving or reviewing.